features/step_definitions/admin_steps.rb

```diff
@@ -70,6 +70,20 @@ def update_settings(settings)
   Language.find_or_create_by_short_and_name("FR", "Francais")
 end
 
+Given /^caching is enabled$/ do
```

Comment:
Maybe take them out for now? We've kind of been mumbling about perhaps getting rid of these settings.